test/test_ops.py

```diff
@@ -1569,5 +1569,42 @@ def test_jit(self, alpha, gamma, reduction, device, dtype, seed) -> None:
         torch.testing.assert_close(focal_loss, scripted_focal_loss, rtol=tol, atol=tol)
 
 
+class TestCIOULoss:
+    @pytest.mark.parametrize("dtype", [torch.float32, torch.half])
+    @pytest.mark.parametrize("device", cpu_and_gpu())
+    def test_ciou_loss(self, dtype, device):
+        box1 = torch.tensor([-1, -1, 1, 1], dtype=dtype, device=device)
+        box2 = torch.tensor([0, 0, 1, 1], dtype=dtype, device=device)
+        box3 = torch.tensor([0, 1, 1, 2], dtype=dtype, device=device)
+        box4 = torch.tensor([1, 1, 2, 2], dtype=dtype, device=device)
+
+        box1s = torch.stack(
+            [box2, box2],
+            dim=0,
+        )
+        box2s = torch.stack(
+            [box3, box4],
+            dim=0,
+        )
+
+        def assert_ciou_loss(box1, box2, expected_output, reduction="none"):
+
+            output = ops.complete_box_iou_loss(box1, box2, reduction=reduction)
+            expected_output = torch.tensor(expected_output, device=device, dtype=dtype)
```

Comment:
```suggestion
            expected_output = torch.tensor(expected_output, device=device)
```

This will automatically take the dtype. You don't need to parameterize here.